### PR TITLE
feat: SAP Cloud Logging as user provided service

### PR DESCRIPTION
**Pull request by qby-ankul**
The plugin currently does not support UPS for Cloud Logging. This is described in the documentation:

https://help.sap.com/docs/cloud-logging/cloud-logging/ingest-via-cloud-foundry-runtime?locale=en-US&version=Cloud#procedures

UPS for Cloud Logging is enabled with this extension.

### CODE REVIEW

File: lib/tracing/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const { registerInstrumentations } = require('@opentelemetry/instrumentation')
 const { BatchSpanProcessor, SimpleSpanProcessor, SamplingDecision } = require('@opentelemetry/sdk-trace-base')
 const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node')
 
-const { getDynatraceMetadata, getCredsForDTAsUPS, augmentCLCreds, _require } = require('../utils')
+const { getDynatraceMetadata, getCredsForDTAsUPS, augmentCLCreds, _require, getCredsForCLSAsUPS } = require('../utils')
```
Comment (sjvans) [inline lib/tracing/index.js:10]:
```suggestion
const { getDynatraceMetadata, getCredsForDTAsUPS, getCredsForCLSAsUPS, augmentCLCreds, _require } = require('../utils')
```

File: lib/utils.js [hunk 2]
```diff
@@ -96,6 +96,15 @@ function getCredsForDTAsUPS() {
     return vcap['user-provided'].find(b => b.name.match(/dynatrace/)).credentials
 }
 
+function getCredsForCLSAsUPS() {
+  if (!process.env.VCAP_SERVICES) return
+  const vcap = JSON.parse(process.env.VCAP_SERVICES)
+
+  // to support connection via user-provided services, APMs requirement is that the instance name contains "cloud-logging"
```
Comment (sjvans) [inline lib/utils.js:103]:
```suggestion
  // to support connection via user-provided services, the instance name must contain "cloud-logging"
```
not an APM requirement

**Review by sjvans — APPROVED** [review]

**Comment by qby-ankul** [comment]
sorry ... for the re-review

Is there a planned date for the next release?

**Review by sjvans — APPROVED** [review]

**Comment by sjvans** [comment]
> sorry ... for the re-review
> 
> Is there a planned date for the next release?

np. should be available by the end of the week latest.

**Pull request merged**